README

```diff
@@ -887,14 +887,23 @@ IMGMOUNT
               the files available from inside DOSBox.
         none: DOSBox will make no attempt to read the file system on the disk.
               This is useful if you need to format it or if you want to boot
-              the disk using the BOOT command. When using the "none"
-              filesystem, you must specify the drive number (2 or 3,
-              where 2 = master, 3 = slave) rather than a drive letter.
-              For example, to mount a 70MB image as the slave drive device,
+              the disk using the BOOT command.
+
+              When using the "none" filesystem, you must specify a drive number:
+
+              Drive type           Drive number
+              ----------           ------------
+              primary floppy drive      0
+              secondary floppy drive    1
+              primary hard drive        2
+              secondary hard drive      3
```

Comment:
This is an area where legacy documentation exists that may make the change confusing.  While I think most people that understand disk controller technology would probably not be affected, there isn't a real nice way to retroactively fix the terminology.  Something like "Primary" and "Secondary" were sometimes referred to by controller and sometimes had it written directly on the cable itself.  It's unfortunate that they chose master and slave for drives on the same controller because neither of them were in any kind of control situation.  Drive 0 and drive 1 on the controller are more applicable here, but since the drives can be considered as one controller port this should be fine.  The point I'm trying to make is that in some situations it can be confusing using terms that can't be retroactively removed from legacy documentation and generally used terms of the period.  It may be best to leave some references such as, "Formerly called master", or "Formerly called slave" which clues in people to the old naming convention while showing intent to correct potential insensitive or outdated terminology.

Reply:
I think terms such as Primary and Secondary are mainly intended to make them easier to understand, but names like "Master" and "Slave" had been traditionally used, including legacy documents etc as you said. These new terms are supposed to make them more intuitive for new people, although I don't think there is an established standard to call them the new terms yet. I suggest references like "Traditionally known as master/slave" or "also known as master/slave" for make things clearer.

Reply:
The end state of this master/slavel/cable-select jumpering was simply to get a C: if you had one HDD or C: and D: if you had two HDDs. 

The IDE controllers were build into the IDE drive themselves. When a HDD was slave-jumpered, its controller communicated with the IDE-controller on the master-jumpered HDD to determine which drive would send signals down the ribbon. 

Because of this forced coordination between controllers, it was impossible to have only a D: in DOS without also have a C:.

So I wonder if this layer of abstraction (drilling down to master/slave/cable-select/etc..) is even necessary DOSBox?

Reply:
See update here: https://github.com/dosbox-staging/dosbox-staging/pull/1116/commits/b634af5fce7ba309c73a0dca5fa30a279d826f7a